test/unit_Hyperrectangle.jl

```diff
@@ -68,22 +68,18 @@ for N in [Float64, Rational{Int}, Float32]
     # diameter
     @test diameter(h) == norm(N[5., 3.] - N[1., 1.], Inf)
 
-    # alternative constructors
+    # alternative constructor
     c = ones(N, 2)
     r = to_N(N, [0.1, 0.2])
     l = to_N(N, [0.9, 0.8])
     h = to_N(N, [1.1, 1.2])
     H1 = Hyperrectangle(c, r)
-    H2 = Hyperrectangle(center=c, radius=r)
-    H3 = Hyperrectangle(low=l, high=h)
-    @test H1.center == H2.center
-    @test H2.center ≈ H3.center
-    @test H1.radius == H2.radius
-    @test H2.radius ≈ H3.radius
-    @test_throws ArgumentError Hyperrectangle(xyz="zyx")
+    H2 = Hyperrectangle(low=l, high=h)
+    @test H1.center ≈ H2.center
+    @test H1.radius ≈ H2.radius
 
     # Test low and high methods for a hyperrectangle
-    H = Hyperrectangle(center=to_N(N, [-2.1, 5.6, 0.9]), radius=fill(to_N(N, 0.5), 3))
+    H = Hyperrectangle(to_N(N, [-2.1, 5.6, 0.9]), fill(to_N(N, 0.5), 3))
```

Comment:
although not pertinent to this PR, i think we can do

```julia
H = Hyperrectangle(N[-2.1, 5.6, 0.9], fill(N(0.5), 3))
```

Reply:
Sometimes this does not work with `Rational`s, although here with only one decimal place it should work.